src/screens/IndexScreen.js

```diff
@@ -29,20 +29,34 @@ import { usePosition, useTrackScreenViewAsync } from '../hooks';
 
 const { MATOMO_TRACKING } = consts;
 
+const TopFilter = {
+  list: 'list',
+  map: 'listTypes'
```

Comment:
why `listTypes` for `map`? in what way is this wording connected?

Reply:
this should have been `map` already. i somehow missed the rename here.